plugins/api-docs/package.json

```diff
@@ -45,6 +46,7 @@
     "@testing-library/user-event": "^12.0.7",
     "@types/jest": "^26.0.7",
     "@types/node": "^12.0.0",
+    "@types/react": "^16.9",
```

Comment:
see the build notification - this actually should go up in the normal dependencies :)